src/components/Upload.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Dropzone from 'react-dropzone';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+import styles from './Upload.scss';
+
+const renderDropzone = field => (
+  <div>
+    <Dropzone
+      accept="image/*"
+      maxSize="3000"
+      className={styles.overide}
+      name={field.name}
+      onDrop={fileToUpload => field.input.onChange(fileToUpload)}
```

Comment:
Field is actually the props sent from Field component 
Field will always send two props when dealing with components 
 [These are props that Field will pass to your wrapped component. The props provided by redux-form are divided into input and meta objects.](http://redux-form.com/6.0.5/docs/api/Field.md/)
In our case Input.value holds an array of [FILE] https://developer.mozilla.org/en/docs/Web/API/File) Objects. With File objects we can store the uploaded photos